apps/code-examples/src/app/code-examples/lookup/search/basic/demo.component.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { SkyMediaBreakpoints, SkyMediaQueryService } from '@skyux/core';
+import { MockSkyMediaQueryService } from '@skyux/core/testing';
```

Comment:
This type is also marked as "internal". I'm going to work on creating a testing controller for the media query stuff that we can incorporate into our code examples. In the meantime, do you think you can only add tests for things that don't require the media query service?

Reply:
Edit: I went ahead and created the controller this afternoon. Let's wait until it gets merged to proceed on this PR. Take a look (specifically at the search specs): https://github.com/blackbaud/skyux/pull/2802

Reply:
from dms: will fix merge conflicts after pr is merged